src/libraries/System.Text.Json/gen/Resources/Strings.resx

```diff
@@ -141,4 +141,16 @@
   <data name="MultipleJsonConstructorAttributeTitle" xml:space="preserve">
     <value>Type has multiple constructors annotated with JsonConstructorAttribute.</value>
   </data>
+  <data name="MultipleJsonExtensionDataAttributeFormat" xml:space="preserve">
+    <value>Type '{0}' has multiple members annotated with 'JsonExtensionDataAttribute'.</value>
+  </data>
+  <data name="MultipleJsonExtensionDataAttributeTitle" xml:space="preserve">
+    <value>Type has multiple members annotated with JsonExtensionDataAttribute.</value>
+  </data>
+  <data name="DataExtensionPropertyInvalidFormat" xml:space="preserve">
+    <value>The data extension property '{0}.{1}' does not match the required signature of 'IDictionary&lt;string, JsonElement&gt;', 'IDictionary&lt;string, object&gt;' or 'JsonObject'.</value>
```

Comment:
It has to be `IDictionary`? It can't be just `Dictionary`?

Reply:
It has to implement `IDictionary<,>` or be `JsonObject`. Can update the message. It's the same one we have in the reflection serializer.

Reply:
Ah, that makes more sense. The way it is currently written, it sounds like the property's type has to strictly be `IDictionary`.

Reply:
```suggestion
    <value>The data extension property '{0}.{1}' does not match the required signature of IDictionary&lt;string, JsonElement&gt;, IDictionary&lt;string, object&gt; or JsonObject.</value>
```

Reply:
I've updated the exception message. I had a question on quotes in https://github.com/dotnet/runtime/pull/58912#discussion_r706239039.